src/test/fuzz/fuzz.cpp

```diff
@@ -56,6 +56,23 @@ static bool read_stdin(std::vector<uint8_t>& data)
 }
 #endif
 
+#if defined(PROVIDE_FUZZ_MAIN_FUNCTION)
+static bool read_file(fs::path p, std::vector<uint8_t>& data)
+{
+    uint8_t buffer[1024];
+    FILE* f = fsbridge::fopen(p.string(), "rb");
+    if (f == nullptr)
```

Comment:
```suggestion
    if (f == nullptr) {
```

According to style-guide

Reply:
Good catch, I actually went with 

```
if (f == nullptr) return false;
```

if that's alright instead, to be consistent with one liners.